docs/sql-data-sources-load-save-functions.md

```diff
@@ -41,6 +41,11 @@ name (i.e., `org.apache.spark.sql.parquet`), but for built-in sources you can al
 names (`json`, `parquet`, `jdbc`, `orc`, `libsvm`, `csv`, `text`). DataFrames loaded from any data
 source type can be converted into other types using this syntax.
 
+For built-in sources, the available extra configuration options are documented in the API documentation,
```

Comment:
I would just simply say:

Please refer API documentation for available options in built-in sources, for example, `org.apache.spark.sql.DataFrameReader` and `org.apache.spark.sql.DataFrameWriter`.